extensions/wikia/Search/classes/IndexService/AllUnifiedSearch.php

```diff
@@ -49,7 +49,30 @@ public function execute() {
 			}
 		}
 
+		$result['is_disabled_in_global_search'] = $this->getWikiGlobalSearchInclusionStats();
 		return $result;
 	}
 
+	/**
+	 * Calculate flag for deciding to include page in global search.
+	 *
+	 * @return array
+	 */
+	protected function getWikiGlobalSearchInclusionStats() {
+		global $wgExcludeWikiFromSearch;
+		global $wgForceWikiIncludeInSearch;
+
+		if (!empty( $wgExcludeWikiFromSearch )) {
```

Comment:
I'm not sure if formatting is not correct - [here](https://github.com/Wikia/guidelines/tree/master/PHP) in bin directory you can find xml with IDE config